pkgs/applications/science/electronics/linux-gpib/linux-gpib-kernel/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, kernel
+, pahole
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux-gpib";
+  version = "4.3.6";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/linux-gpib/files/linux-gpib%20for%203.x.x%20and%202.6.x%20kernels/${version}/${pname}-${version}.tar.gz";
+    hash = "sha256-Gze4xrvkhEgn+J5Jhrycezjp2uhlD1v6aX0WGv4J2Jg=";
+  };
```

Comment:
```suggestion
  inherit (linux-gpib) pname version src;
```

Reply:
Tried this but receive `error: infinite recursion encountered`.  Can you give more details how can I apply this?

Reply:
https://github.com/NixOS/nixpkgs/blob/d85f641287e4f632e9ee3708f0af6d8a41380e36/pkgs/top-level/python-packages.nix#L4773-L4775
This is an example from python3Packages, but you can use `inherit (pkgs) linux-gpib;` in linux-kernels.nix like this.